sdkit/models/model_loader/hypernetwork/__init__.py

```diff
@@ -22,21 +31,24 @@ def load_model(context: Context, **kwargs):
         for size, sd in state_dict.items():
             if type(size) == int:
                 layers[size] = (
-                    HypernetworkModule(size, sd[0], layer_structure, activation_func, weight_init, add_layer_norm,
-                                    use_dropout, activate_output, last_layer_dropout=last_layer_dropout,
-                                    model=layers, device=context.device),
-                    HypernetworkModule(size, sd[1], layer_structure, activation_func, weight_init, add_layer_norm,
-                                    use_dropout, activate_output, last_layer_dropout=last_layer_dropout,
-                                    model=layers, device=context.device),
+                    HypernetworkModule(
+                        size, sd[i], layer_structure, activation_func,
+                        weight_init, add_layer_norm, use_dropout,
+                        activate_output, last_layer_dropout=last_layer_dropout,
+                        model=layers, device=context.device)
+                    for i in range(0, 1)
                 )
 
         override_attention_context_kv(layers)
 
         return layers
     except:
         log.error(traceback.format_exc())
-        log.error(f'Could not load hypernetwork: {model_path}')
```

Comment:
Can you please explain the thinking behind changing the f-strings to %s?

Reply:
I think that's the reason: https://docs.python.org/3/howto/logging.html#optimization. At least that's why I do too.

Reply:
If we're using expensive function calls, then it makes sense. Otherwise f-strings are preferable since they're clearer to read